backend/src/database/DatabaseSeeder.sql

```diff
@@ -0,0 +1,362 @@
+insert into Users (name, surname, email, password, role_id, username) values ('Minnaminnie', 'Helwig', 'mhelwig1g@github.io', 's1GOYRI', 2, 'mhelwig1g');
+insert into Users (name, surname, email, password, role_id, username) values ('Caresa', 'Panton', 'cpanton1h@webs.com', 'v1K@b(@Pc', 2, 'cpanton1h');
+insert into Users (name, surname, email, password, role_id, username) values ('Mata', 'Hansman', 'mhansman1i@nymag.com', 'r52uE{p', 2, 'mhansman1i');
+insert into Users (name, surname, email, password, role_id, username) values ('Lianna', 'Jime', 'ljime1j@phoca.cz', 't2vPg?', 2, 'ljime1j');
+insert into Users (name, surname, email, password, role_id, username) values ('Seka', 'Lynock', 'slynock1k@wisc.edu', 'l44j?n', 2, 'slynock1k');
+insert into Users (name, surname, email, password, role_id, username) values ('Reilly', 'Mattia', 'rmattia1l@latimes.com', 'o5H9o}z*O', 2, 'rmattia1l');
+insert into Users (name, surname, email, password, role_id, username) values ('Henriette', 'Empringham', 'hempringham1m@blogger.com', 'x4/bZyL%', 2, 'hempringham1m');
+insert into Users (name, surname, email, password, role_id, username) values ('Thacher', 'Jeffrey', 'tjeffrey1n@house.gov', 'm8IG06Act4', 2, 'tjeffrey1n');
+insert into Users (name, surname, email, password, role_id, username) values ('Corbie', 'Shore', 'cshore1o@hc360.com', 'g3E7/i1', 2, 'cshore1o');
+insert into Users (name, surname, email, password, role_id, username) values ('Darb', 'Sprasen', 'dsprasen1p@opensource.org', 'b8I"Wx8}}', 2, 'dsprasen1p');
+insert into Users (name, surname, email, password, role_id, username) values ('Ingelbert', 'Toomer', 'itoomer1q@ucsd.edu', 'j98Vlt`#N5', 2, 'itoomer1q');
+insert into Users (name, surname, email, password, role_id, username) values ('Walton', 'Philliskirk', 'wphilliskirk1r@tmall.com', 'j9Ywt2(', 2, 'wphilliskirk1r');
+insert into Users (name, surname, email, password, role_id, username) values ('Harriot', 'Grahl', 'hgrahl1s@amazon.co.jp', 'b4Hbb|', 2, 'hgrahl1s');
+insert into Users (name, surname, email, password, role_id, username) values ('Filmore', 'Wiggett', 'fwiggett1t@tiny.cc', 'i92@leES', 2, 'fwiggett1t');
+insert into Users (name, surname, email, password, role_id, username) values ('Torey', 'Lanchbury', 'tlanchbury1u@sfgate.com', 'r5IF''`GI', 2, 'tlanchbury1u');
+insert into Users (name, surname, email, password, role_id, username) values ('Catlaina', 'Dalgetty', 'cdalgetty1v@utexas.edu', 'l3sLW', 2, 'cdalgetty1v');
+insert into Users (name, surname, email, password, role_id, username) values ('Carolyn', 'Pechan', 'cpechan1w@surveymonkey.com', 'm2=)Aj1', 2, 'cpechan1w');
+insert into Users (name, surname, email, password, role_id, username) values ('Aurel', 'Deerr', 'adeerr1x@huffingtonpost.com', 'q6oeV', 2, 'adeerr1x');
+
+
+INSERT INTO orders (user_id, order_status, total_price)
+VALUES
+(15, 'in-progress', 35.50),
```

Comment:
za order_status prvo kreiraj enum tako da se onemoguci postavljanje neke nepredvidene vrijednosti
`CREATE TYPE status_enum AS ENUM ('delivered', 'canceled', 'in-progress');`

Reply:
Enum vec postoji pogledajte zadnju bazu